### ISSUE TITLE
feature-autoencoder_pretrain

### ISSUE THREAD

**Issue by mareckste** [mareckste/dp_bts#3]
Train encoder-decoder model and use it's output as input into down-sampling approach.

- create additional channel for train and test data
- store these data
- use them during training and evaluation of down-sampling architecture
  - try different loss functions
  - use batch normalization
  - play with architecture, other parameters and advanced training utilities offered in keras
- generate more data
- fix bloated ram issue while having a higher volume of training data

**Comment by mareckste** [comment]
Solved #5
